libs/logging/src/impl/transports/console.ts

```diff
@@ -55,7 +55,7 @@ export class ConsoleTransport implements ITransport {
 
     const messageParts: [unknown] = [color(log)];
     if (options?.objects) {
-      messageParts.push(options.objects);
+      messageParts.push(this.executionContext !== 'browser' ? JSON.stringify(options.objects) : options.objects);
```

Comment:
We could improve the explicit stringify conversion by checking whether the options.object is of kind string already, if so, don't stringify it.
The best way to do this is to outsource the functionality to a small helper method, so that the ternary operator is not so overloaded.